clients/cli/src/error.rs

```diff
@@ -94,12 +95,13 @@ make_errkind_enum!(
 
     // OS (30s)
     30 => OsPwdMissing(IoError),
-    31 => OsCouldNotCreateDir(String, IoError),
-    32 => OsCouldNotCreateFile(String, IoError),
-    33 => OsCouldNotWriteFile(String, IoError),
-    34 => OsCouldNotDeleteFile(String, IoError),
-    35 => OsInvalidPath(String),
-    36 => OsFileCollision(String),
+    31 => OsCouldNotCreateDir(PathBuf, IoError),
+    32 => OsCouldNotCreateFile(PathBuf, IoError),
+    33 => OsCouldNotWriteFile(PathBuf, IoError),
+    34 => OsCouldNotDeleteFile(PathBuf, IoError),
+    34 => OsCouldNotDeleteFolder(PathBuf, IoError),
+    35 => OsInvalidPath(PathBuf),
+    36 => OsFileCollision(PathBuf),
```

Comment:
@Parth no we didn't consolidate, but we did reformat cli errors in general on the hmdb PR. I think it'd be best to leave them alone (revert back to `String`s) so that merging or rebasing will be easier.

Reply:
Chatted and pathbuf can stay